pages/basket/index.js

```diff
@@ -55,7 +57,8 @@ export default function Basket({ products }) {
         {products.map((product, index) => (
           <div key={product.id}>
             <h2>{product.name}</h2>
-            <p>quantity: {quantity[index]}</p>
+            <p>quantity:{quantity[index]}</p>
+            <p>price:{product.price}</p>
```

Comment:
This line is correct, so it can stay as it is.

(The "product" object here has come from the database, and its property "price" is the price we want to display.)